<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QSplitter" name="splitter">
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTreeView" name="treeView"/>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetMaximumSize</enum>
        </property>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProgramm">
    <property name="title">
     <string>Program</string>
    </property>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuProject">
    <property name="title">
     <string>Project</string>
    </property>
    <addaction name="actionLoadProject"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
   </widget>
   <addaction name="menuProgramm"/>
   <addaction name="menuProject"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewElement"/>
   <addaction name="actionNewPlus"/>
   <addaction name="actionNewMinus"/>
   <addaction name="actionNew_Multiplication"/>
   <addaction name="actionNewDivision"/>
   <addaction name="separator"/>
   <addaction name="actionPlay"/>
  </widget>
  <action name="actionClose">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>PxLinX beenden</string>
   </property>
  </action>
  <action name="actionNewElement">
   <property name="icon">
    <iconset>
     <normaloff>Icons/element32.png</normaloff>Icons/element32.png</iconset>
   </property>
   <property name="text">
    <string>neues Element</string>
   </property>
   <property name="toolTip">
    <string>neues Element einf√ºgen</string>
   </property>
  </action>
  <action name="actionNewConnector">
   <property name="icon">
    <iconset>
     <normaloff>../../icons/conector32.png</normaloff>../../icons/conector32.png</iconset>
   </property>
   <property name="text">
    <string>neuer Konnector</string>
   </property>
   <property name="toolTip">
    <string>neuer Konnector</string>
   </property>
  </action>
  <action name="actionNewPlus">
   <property name="icon">
    <iconset>
     <normaloff>Icons/plus24.png</normaloff>Icons/plus24.png</iconset>
   </property>
   <property name="text">
    <string>New Plus</string>
   </property>
  </action>
  <action name="actionNewMinus">
   <property name="icon">
    <iconset>
     <normaloff>Icons/minus24.png</normaloff>Icons/minus24.png</iconset>
   </property>
   <property name="text">
    <string>new Minus</string>
   </property>
  </action>
  <action name="actionNewDivision">
   <property name="icon">
    <iconset>
     <normaloff>Icons/divide24.png</normaloff>Icons/divide24.png</iconset>
   </property>
   <property name="text">
    <string>new Division</string>
   </property>
  </action>
  <action name="actionNew_Multiplication">
   <property name="icon">
    <iconset>
     <normaloff>Icons/multiply24.png</normaloff>Icons/multiply24.png</iconset>
   </property>
   <property name="text">
    <string>New Multiplication</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="icon">
    <iconset>
     <normaloff>Icons/play24.png</normaloff>Icons/play24.png</iconset>
   </property>
   <property name="text">
    <string>play</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionLoadProject">
   <property name="text">
    <string>Load Project</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
